python/lsst/daf/butler/registry/queries/exprParser/exprTree.py

```diff
@@ -164,15 +170,15 @@ class TimeLiteral(Node):
     value : str
         Literal string value.
```

Comment:
Sad that mypy didn't catch these.  At first I thought it was just because I told it not to check the `ply` package, so it couldn't tell what the original type was but still enforced consistency.  But I tested that theory while fixing this, and it's not even testing consistency.  My guess is that because `astropy` doesn't provide type information, it figures it can't tell that `astropy.time.Time` isn't an alias to `str` or contravariant with it, but that's just a guess.